ts/api/api.ts

```diff
@@ -159,75 +165,78 @@ function verifyOrDetectRenderMode(renderMode: RenderMode|null): RenderMode {
  */
 class OpenYoloApiImpl implements OpenYoloApi {
   static async create(
-      providerUrlBase: string,
-      renderMode: RenderMode|null,
-      areTimeoutsDisabled?: boolean,
-      preloadRequest?: PreloadRequest): Promise<OpenYoloApiImpl> {
+    providerUrlBase: string,
+    renderMode: RenderMode | null,
+    areTimeoutsDisabled?: boolean,
+    preloadRequest?: PreloadRequest): Promise<OpenYoloApiImpl> {
     // Sanitize input.
     renderMode = verifyOrDetectRenderMode(renderMode);
 
     const instanceId = generateId();
     const instanceIdHash = await sha256(instanceId);
     const frameManager = new ProviderFrameElement(
-        document,
-        instanceIdHash,
-        window.location.origin,
-        renderMode,
-        providerUrlBase,
-        preloadRequest);
-
-    let channel: SecureChannel|null = null;
+      document,
+      instanceIdHash,
+      window.location.origin,
+      renderMode,
+      providerUrlBase,
+      preloadRequest);
+
+    let channel: SecureChannel | null = null;
     if (areTimeoutsDisabled) {
       channel = await SecureChannel.clientConnectNoTimeout(
-          window, frameManager.getContentWindow(), instanceId, instanceIdHash);
+        window, frameManager.getContentWindow(), instanceId, instanceIdHash);
```

Comment:
this formatting actually seems incorrect; double-indent is normally used for parameter wrapping :/